packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-internal.js

```diff
@@ -50,6 +50,15 @@ class MultiSelectComboBoxInternal extends ComboBoxDataProviderMixin(ComboBoxMixi
 
   static get properties() {
     return {
+      /**
+       * Total number of items.
+       * @type {number | undefined}
+       */
+      size: {
+        type: Number,
+        notify: true,
```

Comment:
Note: the regular web component does not dispatch `size-changed` event, so I had to override the property here to specify `notify: true` - this is done in order to ensure host element `size` is updated properly.